testimony/__main__.py

```diff
@@ -33,13 +33,19 @@ def parse_args():
         '-j', '--json', action='store_true', help='JSON output')
     parser.add_argument(
         '-n', '--nocolor', action='store_true', help='Do not use color option')
+    parser.add_argument(
```

Comment:
We can use https://docs.python.org/2/library/argparse.html#nargs and have on the command line something like: `testimony --tags t1 t2 t3 t4` or `testimony -t t1 t2 t3 t4`.